frontend/app/src/util/AppNavigation.ts

```diff
@@ -374,10 +229,15 @@ export class AppNavigation {
     mainScriptHash: string,
     sidebarElements: BlockNode | undefined
   ): AppRoot {
-    return this.strategy.clearPageElements(
-      elements,
-      mainScriptHash,
-      sidebarElements
-    )
+    if (this.isMPAv1) {
```

Comment:
Could you leave a comment here why we need this special treatment for MPAv1? Bascially explaining why it matter
to the frontend whether the app contains a `/pages` folder or uses `st.navigation` in the backend